helloworld-manifest.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: helloworld
+  labels:
+    app: helloworld
+  namespace: default
+spec:
+  type: ClusterIP
+  ports:
+  - port: 8080
+  selector:
+    app: helloworld
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: helloworld
```

Comment:
Why don't we use https://github.com/giantswarm/e2e-app as we do in our e2e tests?

Reply:
Because I didn't know it. And now that I know the repo, my eagerness to use a completely undocumented repo is zero.

Reply:
Anyway, I'll use it, as the option to have long-running requests with `/delay/5` will be helpful.

Reply:
Check the e2e tests in `aws-operator`. The draining tests use the e2e app.